src/components/UnitBox.example.tsx

```diff
@@ -0,0 +1,40 @@
+import { useStateValue } from '../state/state';
+import React, { CSSProperties } from 'react';
+import type { UnitBoxes } from '../types';
+
+interface UnitBoxProps {
+	field: keyof UnitBoxes;
+}
+
+/**
+ * This is a temporary component that was made for testing the state
+ * management setup. The pattern used here will be integrated into the
+ * statically framed elements during the upcoming AR -> Data ticket
```

Comment:
Might want to explicitly link the ticket or provide the ticket number for posterity.